test/quantization/test_numeric_suite.py

```diff
@@ -95,8 +98,10 @@ def compare_and_validate_results(float_model, q_model):
             for k, v in weight_dict.items():
                 self.assertTrue(v["float"].shape == v["quantized"].shape)
 
+        # Test static quantization
```

Comment:
This test is too big now, can you refactor into smaller tests: Perhaps one for Conv, one for Linear and one for dynamic quant?